__tests__/app.test.js

```diff
@@ -122,3 +122,30 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("status(200), returns an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((user) => {
+          expect(user).toEqual({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+        expect(body.users.length).toBe(4);
+        expect(Array.isArray(body.users)).toBe(true);
+      });
+  });
+  test("status 404, returns not found when passed a route that does not exist", () => {
```

Comment:
Test not needed, you are doing this test already earlier. This test isnt really to do with this end point specifically